### ISSUE TITLE
Upgrade Qt to 5.5

### ISSUE THREAD

**Issue by jakepetroules** [Homebrew/homebrew#41359]
We just released this a few days ago. ;)

Also,

> There needs to be an OpenSSL dep here ideally, but qt keeps ignoring it.

A new SecureTransport-based SSL backend was added to Qt recently (https://codereview.qt-project.org/#/c/101485). This will become the default in Qt 5.6 for OS X and iOS, so OpenSSL should not be needed at all. You should amend the comment to remove that block once 5.6 is released (or configure with -no-openssl -secure-transport).

**Comment by DomT4** [comment]
We're aware. It has some issues we're working on.

See #41248 & #41252.

**Comment by jakepetroules** [comment]
Excellent, thanks. :+1:

**Comment by DomT4** [comment]
There may be a documentation typo in the `configure` script. Passing `-secure-transport` (Which configure suggests is correct) gets `-secure-transport: invalid command-line switch` but the actual `configure` checks suggests it is supposed to be `-securetransport` which it does seem to like?

```
    securetransport)
        if [ "$VAL" = "yes" ] || [ "$VAL" = "no" ]; then
            CFG_SECURETRANSPORT="$VAL"
        else
            UNKNOWN_OPT=yes
        fi
        ;;
```

**Comment by jakepetroules** [comment]
Indeed. Will be fixed in 5.5.1 - https://codereview.qt-project.org/#/c/120695/
